addon/styles/partials/_uni-dropdown.scss

```diff
@@ -17,6 +17,10 @@
     }
   }
 
+  .uni-dropdown__button--error {
+    border-color: color('feedback', 'error');
```

Comment:
From what I've searched, we normally use single quotes.

https://github.com/uniplaces/ember-cli-uniq/search?p=2&q=color%28%22&type=&utf8=%E2%9C%93